site/content/protocols/survey/0.1/readme.md

```diff
@@ -0,0 +1,179 @@
+
+### Roles
+
+There are two parties in a typical survey interaction. The first party, `requester`, issues the survey with its schemata and the second party, `responder`, responds with the answer-data. The parties must have already exchanged pairwise keys and created a connection. These keys can be used to encrypt and verify the response. When the response is received by the requestion, the requestor can know with a high level of certainty that it was sent by responder.
+
+In this tutorial Alice (the `requester`) initiates the interaction, Alice creates the survey and sends it to Bob. The survey includes the questions, ui, initial data and i18n data.
+
+In this tutorial, Bob (the `responder`) receives the request and must respond to the survey (or decline it).
+
+### States
+
+This protocol follows the request-response protocol style and only requires the simple state of producing a conversational message and waiting for a response.
+
+The `abandoned` and `completed` states are terminal states and there is no expectation that the protocol can be continued after reaching those states.
+
+Either party may send a problem-report message earlier in the flow to terminate the protocol before its normal conclusion.
+
+#### States for Requester
+
+The `requester` agent goes through the following states:
+- request-sent
+- completed
+
+The state transition table for the `requester` is:
+
+State/Event|Send Request|Receive Response|Send or Receive Problem Report
+---|---|---|---
+*start*|Transition to **request-sent**| | 
+request-sent| | Transition to **completed**|Transition to **abandoned**
+completed| | 
+abandoned | | 
+
+#### States for Responder
+
+The `responder` agent goes through the following states:
+- request-received
+- completed
+
+The state transition table for the `responder` is:
+
+State/Event|Receive Request|Send Response|Send or Receive Problem Report
+---|---|---|---
+*start*|Transition to **request-received**| |
+request-received| | Transition to **completed** | Transition to **abandoned**
+completed | | | 
+abandoned | | | 
+
+After receiving a survey-request, the responder may send a problem-report to the requestor using the information in the request to decline (abandon) the protocol. 
+
+### Messages
+### Survey Request Message
+
+The protocol begins when the `requester` sends a `survey-request` message to the responder:
+
+DIDComm V1 Example:
+```json
+{
+    "@type": "https://didcomm.org/survey/0.1/request",
+    "@id": "8192855c-89f3-5bb5-4971-7be10cbc6c71",
+    "~thread": {
+        "thid": "5689db78-5123-2aad-448d-0203107fee11"
+    },
+    "request": {
+        "survey_id": "750d9731-562b-f8a9-48df-89b12a1ec7f3",
+        "survey_schema": "json string defining the survey schema with the questions, answers and validations",
+        "ui_schema": "json string defining the ui schema such as layout, controls, help text to aid rendering of the survey",
```

Comment:
I've change the Survey Request Message to JSON objects (instead of JSON strings) and added an example for each of the JSON objects in the request message as well as a full example message.